website/docs/prysm-usage/slasher.md

```diff
@@ -3,71 +3,41 @@ id: slasher
- - Double voting
-   - occurs when a validator signs two different beacon blocks in the same epoch
- - Surround votes
-   - occurs when a validator signs two conflicting attestations  
+The main purpose of slashing is to mitigate attacks that might be otherwise cheap to perform on the eth2 network, such as trivially creating conflicting forks of validators attesting on a different view of historical checkpoints.
+A validator that correctly follows the protocol never emits a slashable vote in normal operations. Validators will not be slashed for simply being offline.
 
-#### Block Proposer slashings
- - occurs when a proposer broadcasts more than one block for the same slot
- 
-![Prysm with Slasher](/img/prysm-with-slasher.png)
+## What is a Slasher
 
-## Get Prysm and start beacon-chain
-> **NOTICE:** If beacon-chain is already running, skip to step 2 
+**Slasher** is the name of software that can detect slashable events from validators and report those to the protocol. You can think of a slasher as the “police” of the network. Running a slasher is **optional**. In order to detect slashable messages, the slasher records the attesting and proposing history for every validator on the network, and cross references this history with what is broadcasted to find slashable messages such as double blocks or surrounding votes.
 
-1. To begin, follow the instructions for [GNU\Linux](/docs/install/linux), [macOS](/docs/install/mac), [ARM64](/docs/install/arm), or [Windows](/docs/install/windows) to fetch and install Prysm.  The beacon-chain process should be running and fully synced before proceeding.
+All the network needs is *1 honest, properly functioning slasher* to monitor the network because any slashings found are propagated to the entire network for it to be put into a block as soon as possible.
 
-## Run Slasher
+Running a slasher is not meant to be profitable. Slashing is meant to be rare and whistleblower rewards are low on purpose. Running a slasher is meant to be an *altruistic action*, and once again, only a single honest, properly functioning slasher needs to be active in the network to catch slashable offenses. Thankfully, this is a low bar to entry, and we envision quite a lot of users and entities will run slashers to ensure network security.
 
-2. Depending on your platform, issue the appropriate command alongside any [startup parameters](/docs/prysm-usage/parameters#slasher-parameters) from the examples below to start slasher.
+## Running Slasher
 
-#### Running slasher with prysm.bat on Windows
+Running a slasher is as simple as adding the `--slasher` flag to your **beacon node**. Doing this will power up your beacon node to perform slashing detection. Slasher is very heavy on database access and disk usage, so you might see it using over 70Gb of extra storage on disk than normal on mainnet. Given slasher needs to store a lot of information about attestations and blocks happening in the network, this is to be expected.
```

Comment:
```suggestion
Running a slasher is as simple as adding the `--slasher` flag to your **beacon node**. Doing this will power up your beacon node to perform slashing detection. Slasher is very heavy on database access and disk usage, so you might see it using over 70Gb of extra storage on disk on mainnet. Given slasher needs to store a lot of information about attestations and blocks happening in the network, this is to be expected.
```

I think the sentence is easier to understand this way. Feel free to ignore this suggestion.